### PR TITLE
Fix broken dependencies

### PR DESCRIPTION
**Pull request by feup-jmc**
More details in #1

### CODE REVIEW

File: d4rl/setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     install_requires=[
         "gym",
         "numpy",
-        "mujoco_py",
+        "mujoco-py",
```
Comment (mihdalal) [inline d4rl/setup.py:11]: Why is this necessary? The install runs fine without this. (same for all replacements of mujoco_py with mujoco-py.

**Comment by feup-jmc** [comment]
Closing this PR as a18cebd96b5257fb89bfa28118d871287b8f005f has already solved dependency issues.
